src/test/java/com/sonymobile/tools/gerrit/gerritevents/workers/EventThreadTest.java

```diff
@@ -77,6 +78,7 @@ public static void beforeClass() {
      * @throws Exception if so.
      */
     @Test(timeout = 1000)
+    @Ignore
```

Comment:
Probably better to just remove these tests

Reply:
gone